lib/manageiq/rpm_build/build_hotfix.rb

```diff
@@ -64,6 +65,11 @@ def update_spec
         File.write(rpm_spec, spec_text)
       end
 
+      def clean_hotfix_directory
+        files = Dir.glob("*").reject {|path| path.end_with?("src.rpm") }
```

Comment:
Minor
```suggestion
        files = Dir.glob("*").reject { |path| path.end_with?("src.rpm") }
```

Reply:
If we're not using the glob part of Dir.glob would Dir.children(".") work?

Reply:
Event better:

```suggestion
        files = Dir.glob("*[!.src.rpm]")
```

Reply:
haha, I had glob negation at first but thought this looked more obvious considering performance isn't going to matter on a handful of file.  I can change it if you also liked it 😉

Reply:
haha, ship it... just verifying it

Reply:
ok, testing looks fine